examples/ovsbridge_setup.yml

```diff
@@ -3,6 +3,14 @@ interfaces:
   - name: eth1
     type: ethernet
     state: up
+  - name: ovs0
```

Comment:
AFAIU you agreed to use `create` instead of `setup`. And I would still prefer to keep `eth1` in the name.

Reply:
Right, will change it to `create`.
Regarding the iface name, I found having `_eth1_ovs0_` very unhelpful and even confusing. If I'll add another port later on, I'll need to mention it as well. I also could add the type and other details.
If we keep up with the example, I think we will add to each file example multiple stuff and detailing it in the name makes little sense.

Reply:
I do not see much value with mentioning ovs0 since it is a new interface that we create and that it is part of a bridge is just an implementation detail. Actually, maybe ovs0 should be renamed to nmstate_ovs or similar to ensure there are no conflicts with existing devices.